sample-apps/go-wrapper-aws-sdk-terraform/layer.tf

```diff
@@ -0,0 +1,20 @@
+locals {
+  collector_layer_arns = {
+    "ap-northeast-1" = "arn:aws:lambda:ap-northeast-1:901920570463:layer:aws-otel-collector-ver-0-29-1:1"
```

Comment:
I think these are out of date? You'll probably want to update them to be `36` and match .NET for the latest? Although I'm not sure what to do when a layer has never been released as is the case with "go" right now...

.NET currently uses `36`:

https://github.com/aws-observability/aws-otel-lambda/blob/cd6d8ba2fb8bc9b5b323dbc70f86c7529554ae7d/sample-apps/dotnet-wrapper-aws-sdk-terraform/layer.tf#L1-L6

We know `36` exists so it should be find to use...